ADBench/run-all.ps1

```diff
@@ -189,12 +189,7 @@ Class Tool {
 		$cmd = ""
 		$cmdargs = @($dir_in, $dir_out, $fn, $script:nruns_f, $script:nruns_J, $script:time_limit)
 		if ($this.type -eq "bin") {
-			if ($this.name -eq "DiffSharp") {
-				$builddir = ""
-			} else {
-				$builddir = "\$script:buildtype"
-			}
-			$cmd = "$script:bindir\tools\$($this.name)$builddir\Tools-$($this.name)-$objective.exe"
+			$cmd = "$script:bindir\tools\$($this.name)\Tools-$($this.name)-$objective.exe"
```

Comment:
@awf: Can you check this still works for you?  On my machine the tools don't get put inside a subdirectory named after their build type.

Reply:
Apparently this was addressed by https://github.com/awf/ADBench/pull/38.  It has been closed though I'm not sure why.